blackbox/docs/sql/system.txt

```diff
@@ -922,23 +922,21 @@ The table schema is the following:
 |                  | check for the setting has passed.|             |
 +------------------+----------------------------------+-------------+
 
-Currently Crate checks the following:
-
-+------------------+------------------------------------+
-|       id         |           Check name               |
-+==================+====================================+
-| 1                | Minimum master nodes.              |
-+------------------+------------------------------------+
-| 2                | Recovery expected nodes.           |
-+------------------+------------------------------------+
-| 3                | Recovery after nodes.              |
-+------------------+------------------------------------+
-| 4                | Recovery after time.               |
-+------------------+------------------------------------+
-| 5                | Number of partitions.              |
-+------------------+------------------------------------+
-| 6                | Java 7 deprecation warning.        |
-+------------------+------------------------------------+
+Example output::
```

Comment:
selecting only id and desc would make it more readable imho.
probably example query is better than example output